akka-stream-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -14,7 +14,6 @@ object Timeouts {
 
   def publisherShutdownTimeoutMillis: Int = 1000
 
-  def defaultTimeoutMillis(implicit system: ActorSystem): Int =
-    500.millis.dilated(system).toMillis.toInt
+  def defaultTimeoutMillis: Int = 500
```

Comment:
we have no system at construction time, and I didn't consider it important enough to solve that, perhaps I can override this in ActorSystemLifecycle